tox.ini

```diff
@@ -1,14 +1,11 @@
 [tox]
 envlist =
-    # Python 2.7 support was removed in Sphinx 2
-    py{27}-sphinx{17,18}{-html4,-html5,}{-qa,}
-    py{36,37,38,39}-sphinx{18,20,21,22,23,24,30,31,32,33,34,35,40,41,42,43,44,45,50,51,52}{-html4,-html5,}{-qa,}
-    # Python 3.10 working from Sphinx 4.2 and up
-    py{310}-sphinx{42,43,44,45,50,51,52,53,latest}{-html4,-html5}{-qa,}
-    # Sphinx 6+ has simplified docutils and Python support
-    py{38,39,10}-sphinx{60,61,70,71}{-html5,}{-qa,}
+    py{36,37,38,39,310}-sphinx{50,51,52,53}{-html4,-html5}
+    py{38,39,310}-sphinx{60,61,62}{-html4,-html5}
```

Comment:
I suppose we haven't talked much about html4 just yet.

Technically, html4 is already deprecated in the current release, and was planned to throw a hard error in our next release. So, that could be part of this release too, and in both cases, I'd probably say it's fine to stop testing the html4 writer, visually and in CI.

We'd probably leave the code around for now, but it could be removed any time after this release and when we support Sphinx 7+.

Anyways, that's all to say, should we only test on html5 writer here?

Reply:
Also, I'd leave the `-qa` postfix targets. The docs mention these targets in verifying pull request changes:

https://sphinx-rtd-theme.readthedocs.io/en/stable/contributing.html#testing

Reply:
> Technically, html4 is already deprecated in the current release, and was planned to throw a hard error in our next release. So, that could be part of this release too, and in both cases, I'd probably say it's fine to stop testing the html4 writer, visually and in CI.

I'm happy to remove it in _this 2.0 release_ as it was planned, actually 👍🏼